ui_backend/pkg/rest/controllers.go

```diff
@@ -79,15 +80,15 @@ func createRegionFindingsFromTargets(targets *backendmodels.Targets) []models.Re
 	return items
 }
 
-func getTargetLocation(target backendmodels.Target) (string, error) {
+func getTargetRegion(target backendmodels.Target) (string, error) {
 	discriminator, err := target.TargetInfo.ValueByDiscriminator()
 	if err != nil {
 		return "", fmt.Errorf("failed to get value by discriminator: %w", err)
 	}
 
 	switch info := discriminator.(type) {
 	case backendmodels.VMInfo:
-		return info.Location, nil
+		return strings.Split(info.Location, "/")[0], nil
```

Comment:
This is assuming you know how the location was created, and true only for AWS, is this safe/right to get the region?
Maybe better to also save the region in a separate value on the target if it is needed? Also, region is for AWS, what about other cloud-providers? I think we call it  `getTargetLocation` to be generic.

Reply:
This dashboard is specific to AWS. We can discuss if we want to use general terminology there.

Reply:
if the widget is AWS specific maybe we should add it to the api path name or description

Reply:
Need to move this to a function (with unit test). Need to do the split only for AWS provider, for other providers need to just return the location